mysql/models/user.rb

```diff
@@ -0,0 +1,16 @@
+class User
+  attr_accessor :email, :password, :updated_at
```

Comment:
(YAGNIの観点)

これらもattr_readerでいいんじゃないかな？
```suggestion
  attr_reader :email, :password, :updated_at
```